[spdlog] arm-linux build failure

**Host Environment**
- OS: [Ubuntu 18.04]
- Compiler: gcc

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install spdlog --triplet arm-linux`

**Failure logs** 
Starting package 1/1: spdlog:arm-linux
Building package spdlog[core]:arm-linux...
-- Using cached /home/mohsen/vcpkg/downloads/gabime-spdlog-v1.3.1.tar.gz
-- Using source at /home/mohsen/vcpkg/buildtrees/spdlog/src/v1.3.1-fef0d0df4a
-- Configuring arm-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: /home/mohsen/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/bin/cmake /home/mohsen/vcpkg/buildtrees/spdlog/src/v1.3.1-fef0d0df4a -DSPDLOG_FMT_EXTERNAL=ON -DSPDLOG_BUILD_BENCH=OFF -DCMAKE_MAKE_PROGRAM=/opt/fslc-wayland/2.7/sysroots/x86_64-fslcsdk-linux/usr/bin/ninja -DCMAKE_SYSTEM_NAME=Linux -DBUILD_SHARED_LIBS=OFF -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/opt/fslc-wayland/2.7/sysroots/x86_64-fslcsdk-linux/usr/share/cmake/OEToolchainConfig.cmake -DVCPKG_TARGET_TRIPLET=arm-linux -DVCPKG_SET_CHARSET_FLAG=ON -DVCPKG_PLATFORM_TOOLSET=external -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE -DCMAKE_VERBOSE_MAKEFILE=ON -DVCPKG_APPLOCAL_DEPS=OFF -DCMAKE_TOOLCHAIN_FILE=/home/mohsen/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON -DVCPKG_CXX_FLAGS= -DVCPKG_CXX_FLAGS_RELEASE= -DVCPKG_CXX_FLAGS_DEBUG= -DVCPKG_C_FLAGS= -DVCPKG_C_FLAGS_RELEASE= -DVCPKG_C_FLAGS_DEBUG= -DVCPKG_CRT_LINKAGE=dynamic -DVCPKG_LINKER_FLAGS= -DVCPKG_TARGET_ARCHITECTURE=arm -DCMAKE_INSTALL_LIBDIR:STRING=lib -DCMAKE_INSTALL_BINDIR:STRING=bin -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=/home/mohsen/vcpkg/packages/spdlog_arm-linux/debug
    Working Directory: /home/mohsen/vcpkg/buildtrees/spdlog/arm-linux-dbg
    Error code: 1
    See logs for more information:
      /home/mohsen/vcpkg/buildtrees/spdlog/config-arm-linux-dbg-out.log
      /home/mohsen/vcpkg/buildtrees/spdlog/config-arm-linux-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:305 (vcpkg_execute_required_process)
  ports/spdlog/portfile.cmake:20 (vcpkg_configure_cmake)
  scripts/ports.cmake:91 (include)


Error: Building package spdlog:arm-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: spdlog:arm-linux
  Vcpkg version: 2019.09.12-unknownhash

cat  /home/mohsen/vcpkg/buildtrees/spdlog/config-arm-linux-dbg-err.log
CMake Error at /home/mohsen/vcpkg/scripts/buildsystems/vcpkg.cmake:256 (_find_package):
  Could not find a package configuration file provided by "fmt" with any of
  the following names:

    fmtConfig.cmake
    fmt-config.cmake

  Add the installation prefix of "fmt" to CMAKE_PREFIX_PATH or set "fmt_DIR"
  to a directory containing one of the above files.  If "fmt" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:59 (find_package)




**Additional context**
i am Able to build spdlog with triplet x86-linux
a also built fmt, protobuf, asio whit this configuration

with following config in arm-linux 

set(VCPKG_TARGET_ARCHITECTURE arm)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE static)

set(VCPKG_CMAKE_SYSTEM_NAME Linux)

set(CMAKE_MODULE_PATH /home/mohsen/vcpkg/installed/arm-linux/share/)
set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE /opt/fslc-wayland/2.7/sysroots/x86_64-fslcsdk-linux/usr/share/cmake/OEToolchainConfig.cmake)


failed 

./vcpkg install spdlog --triplet arm-linux
The following packages will be built and installed:
    spdlog[core]:arm-linux
Starting package 1/1: spdlog:arm-linux
Building package spdlog[core]:arm-linux...
CMake Error at scripts/cmake/vcpkg_common_functions.cmake:1 (include):
  include could not find load file:

    execute_process
Call Stack (most recent call first):
  scripts/ports.cmake:90 (include)


Error: Building package spdlog:arm-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: spdlog:arm-linux
  Vcpkg version: 2019.09.12-unknownhash

Additionally, attach any relevant sections from the log files above.

--- a/scripts/buildsystems/vcpkg.cmake
+++ b/scripts/buildsystems/vcpkg.cmake
@@ -230,6 +230,16 @@ else() #Release build: Put Release paths before Debug paths. Debug Paths are req
     )
 endif()
 
+# If one CMAKE_FIND_ROOT_PATH_MODE_* variables is set to ONLY, to  make sure that ${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET} 
+# and ${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/debug are searched, it is not sufficient to just add them to CMAKE_FIND_ROOT_PATH,
+# as CMAKE_FIND_ROOT_PATH specify "one or more directories to be prepended to all other search directories", so to make sure that 
+# the libraries are searched as they are, it is necessary to add "/" to the CMAKE_PREFIX_PATH
+if(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE STREQUAL "ONLY" OR 
+   CMAKE_FIND_ROOT_PATH_MODE_LIBRARY STREQUAL "ONLY" OR
+   CMAKE_FIND_ROOT_PATH_MODE_PACKAGE STREQUAL "ONLY")
+   list(APPEND CMAKE_PREFIX_PATH "/")
+endif()
+
 set(VCPKG_CMAKE_FIND_ROOT_PATH ${CMAKE_FIND_ROOT_PATH})
 
 file(TO_CMAKE_PATH "$ENV{PROGRAMFILES}" _programfiles)

